sdk/communication/Azure.Communication.PhoneNumbers/api/Azure.Communication.PhoneNumbers.netstandard2.0.cs

```diff
@@ -17,6 +17,11 @@ namespace Azure.Communication.PhoneNumbers
         public static bool operator !=(Azure.Communication.PhoneNumbers.BillingFrequency left, Azure.Communication.PhoneNumbers.BillingFrequency right) { throw null; }
         public override string ToString() { throw null; }
     }
+    public partial class DiagnosticPhoneNumbersClientOptions : Azure.Communication.PhoneNumbers.PhoneNumbersClientOptions
```

Comment:
Why do we need this in the public API? Cannot we just add this type to our test project? Client constructors allow passing custom client options that we can use